### PR TITLE
send mail body as actual body instead of inline attachment

### LINKED ISSUE #55: E-mail is sent as "inline" attachment instead of regular mail body
Whenever I send an e-mail using `blastula::smtp_send()` the actual message is sent as an "inline attachment" which – at least in Thunderbird 60 with default settings – results in no (visual) text in the mail body unless you [explicitly activate `Display Attachments Inline`](https://kb.wisc.edu/page.php?id=40875).

In the [code of `smtp_send.R`](https://github.com/rich-iannone/blastula/blob/c33786c546c2a6205cc4b7fb43f107528c9266c5/R/smtp_send.R#L250) I see that [mailsend-go's `attach` option](https://github.com/muquit/mailsend-go#synopsis) is used with the `-inline` flag. I guess it would be better to use the `body` option instead (at least by default), so that all mail clients can properly display the message.

Or is there a specific reason why the `attach` option is used?

---

_I'm using the latest dev version of blastula on Ubuntu Linux 16.04 with latest mailsend-go v1.0.4._

### PR DESCRIPTION
**Pull request by salim-b**
~~and rely on mailsend-go's auto-detection of the MIME type~~

This fixes #55

### CODE REVIEW

**Comment by rich-iannone** [comment]
Thanks for making this PR. The reason for my hesitance before was due to some earlier problems with supplying a long string directly to `body`. Also, echoing the command line input via `echo = TRUE` will produce a huge string (which could be truncated, just below https://github.com/rich-iannone/blastula/blob/master/R/smtp_send.R#L269).

With this change, were you able to successfully send test emails that looked as expected?

**Comment by salim-b** [comment]
> With this change, were you able to successfully send test emails that looked as expected?

Yes :)

(I'm using mailsend-go v1.04 on Ubuntu 16.04)

> (...) some earlier problems with supplying a long string directly to `body`. Also, echoing the command line input via echo = TRUE will produce a huge string (which could be truncated (...)

Well, I pass the message to mailsend-go's `body` option as `-file`, so the string isn't any longer than it was before with the `attach -file` option.

When I use `blastula::smtp_send(echo = TRUE)` I get output like the following:

```
The command for sending the email message is:

/usr/local/bin/mailsend-go -sub Testing -***.***.ch -port 587 auth -user ***@***.ch -pass ***** -fname Salims MailBot 🤖 -from ***@***.ch -to ***@***.ch body -file /tmp/RtmpP8jiW5/file635241705813.html

The email message was sent successfully.
```

File: R/smtp_send.R [hunk 1]
```diff
@@ -249,10 +249,8 @@ smtp_send <- function(email,
       `-to` = to,
       `-cc` = cc,
       `-bcc` = bcc,
-      `attach` = no_options(),
-      `-file` = tempfile_,
-      `-mime-type` = "text/html",
-      `-inline` = no_options()
+      `body` = no_options(),
+      `-file` = tempfile_
```
Comment (rich-iannone) [inline R/smtp_send.R]:
```suggestion
      `-file` = tempfile_, 
      `-mime-type` = "text/html"
```

Comment (rich-iannone) [reply R/smtp_send.R]: @jcheng5 suggested that we make the mime-type explicit.

Comment (salim-b) [reply R/smtp_send.R]: > @jcheng5 suggested that we make the mime-type explicit.

Ok.

**Comment by rich-iannone** [comment]
@salim-b Thanks for your work on this. Sorry to say that we can't use this PR because `mailsend-go` is no longer used in **blastula**. Going to close.

**Comment by salim-b** [comment]
> @salim-b Thanks for your work on this. Sorry to say that we can't use this PR because `mailsend-go` is no longer used in **blastula**. Going to close.

Yeah, I noticed the switch to `curl::send_mail()`, which I think makes sense (though I haven't tested it yet).

I guess you're also going to abandon https://github.com/rich-iannone/blastula/pull/86 then? Or would you still accept external dependencies if they're completely optional as it is the case with using `minify()`?

**Pull request closed without merging**